IFC4x3/Properties/p/PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv" Name="PuttingIntoOperationDate" UniqueId="0399d1df-3ef2-429f-a231-050678d21739" PrimaryDataType="IfcDate" />
```

Comment:
I would propose a renaming to `StartOperationDate`. And a question, does this really fit into Manufacturer Occurence? Looking forward to a discussion.

Reply:
This one is tentatively removed from this pull request. All these "XxxDate" will be managed together in another pull request.